scripts/prow/README.adoc

```diff
@@ -1,110 +1,147 @@
 = Creating Prow jobs for openshift-docs branches
 
-Use the `add-new-prow-config.sh` script to populate the `$HOME/release/ci-operator/config/openshift/openshift-docs` directory with a new Prow job branch configuration.
+To populate the `$HOME/release/ci-operator/config/openshift/openshift-docs` directory with a new Prow job branch configuration, use the `add-new-prow-config.sh` script.
+
 The script uses the `./openshift-openshift-docs-BRANCH.yaml` template file to create the new job config.
-After you create the job config, you run the CI make jobs to create the Prow jobs.
-Then, open a pull request against the https://github.com/openshift/release repository to make the Prow job live for that branch.
+
+To create the Prow jobs, you must run the CI make jobs. Then, open a pull request against the https://github.com/openshift/release repository to make the Prow job live for that branch.
 
 .Prerequisites
 
-. Install link:https://podman.io/docs/installation[Podman].
+. You have installed link:https://podman.io/docs/installation[Podman].
+
+. You have forked the https://github.com/openshift/release repository.
++
+For more information on how to fork a repository, see link:https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo[Fork a repository] (GitHub documentation).
 
-. Fork and clone the https://github.com/openshift/release repository to `$HOME/release` and create a new branch.
+. You have cloned the OpenShift release repository to `$HOME/release`:
 +
-[NOTE]
+[IMPORTANT]
 ====
-If you have already forked the repository, make sure it is up to date.
-Run the following commands:
+If you have already cloned the repository, ensure that it is up to date by running the following commands:
 
 [source,terminal]
 ----
-cd $HOME/release
-git remote add upstream git@github.com:openshift/release.git
-git checkout master
-git fetch upstream
-git rebase upstream/master
-git checkout <your_new_branch> <1>
+$ cd $HOME/release
```

Comment:
Because this isn't getting published to the web, I would recommend backing this change out and leaving it as it is in the previous version. Also not sure why you removed the first 2 lines. 

My suggestion is to remove all the changes up to "Procedure" because it's actually confusing me more to have the instructions for updating the repo, *then* cloning it.